nupic/research/frameworks/vernon/trainables.py

```diff
@@ -45,7 +45,7 @@
 class RayActorHelperMethods:
     def get_node_ip(self):
         """Returns the IP address of the current node."""
-        return socket.gethostbyname(socket.getfqdn())
+        return socket.gethostbyname(socket.gethostname())
```

Comment:
I am not entirely sure, but I think Marcus replaces these methods with methods from Ray in the pending PRs.

Reply:
It was merged 2 days ago https://github.com/numenta/nupic.research/pull/374